pkg/types/baremetal/validation/platform.go

```diff
@@ -84,6 +105,18 @@ func ValidatePlatform(p *baremetal.Platform, n *types.Networking, fldPath *field
 	if err := validateIPNotinMachineCIDR(p.BootstrapProvisioningIP, n); err != nil {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("bootstrapHostIP"), p.BootstrapProvisioningIP, err.Error()))
 	}
+	if p.BootstrapOSImage != "" {
+		//if err := validate.URI(p.BootstrapOSImage); err != nil {
```

Comment:
I think we should drop the commented code.

Reply:
+1.